src/libpatch/analysis/mod.rs

```diff
@@ -2,25 +2,19 @@ use std::io::{self, Write};
 use std::vec::Vec;
 
 use crate::modified_file::ModifiedFile;
-use crate::patch::{
-    FilePatchApplyReport,
-    TextFilePatch,
-    PatchDirection,
-};
+use crate::patch::{FilePatchApplyReport, PatchDirection, TextFilePatch};
```

Comment:
I understand this is what `cargo fmt` does, but I'm not quite sure the re-formatted code is indeed better. Especially since code changes are still shown as diff, which is line-based…